### ISSUE TITLE
Instruction to add line to bashrc not showing up when installing from package

### ISSUE THREAD

**Issue by blackbrokkoli** [wting/autojump#568]
When I install autojump manually, it supplies a warning:

`Please manually add the following line(s) to ~/.bashrc:`
`[[ -s /home/USER/.autojump/etc/profile.d/autojump.sh ]] && source /home/USER/.autojump/etc/profile.d/autojump.sh`
`Please restart terminal(s) before running autojump.`

Using autojump only works after doing these stepts (duh). 

This is not a problem, but when I install it via `apt` it just fails silently. Means using `j` just fails with the usual "unknown command"-dialogue and leaves no clue for the user how to fix it. Tested unsystematically on Linux Mint and Ubuntu.

**Comment by wting** [comment]
Yeah, it's something that should be fixed in in the Debian package but that's being maintained by someone else so I'm not exactly sure how to do display post installation messages when installing via `apt-get` or `aptitude`. 😬
